bmq-sdk/src/test/java/com/bloomberg/bmq/it/TcpBrokerConnectionIT.java

```diff
@@ -608,18 +606,16 @@ public void testNegotiationMpsEx() throws IOException {
         final TestTcpServer[] servers =
                 new TestTcpServer[] {
                     new BmqBrokerSimulator(opts.brokerUri().getPort(), Mode.BMQ_AUTO_MODE),
-                    BmqBrokerTestServer.createStoppedBroker(opts.brokerUri().getPort()),
                     BmqBrokerContainer.createContainer(opts.brokerUri().getPort())
                 };
 
         assertFalse(servers[0].isOldStyleMessageProperties());
         assertFalse(servers[1].isOldStyleMessageProperties());
-        assertTrue(servers[2].isOldStyleMessageProperties());
```

Comment:
Does it mean we lost possibility to test old style properties?

Reply:
We have subscriptions enablement planned, so in the near future we should switch for new message properties everywhere in the new versions of SDKs. Also as far as I remember, for Java SDK we are good to proceed and switch to new message properties fully. In general, old style properties / backward compatibility should be tested somewhere near the broker in C++ repo I think. @quarter-note is it correct?

Reply:
Discussed it with @quarter-note 
Some points:
- We do not change SDK implementation code related to message properties in this PR
- Changes affect ITs only
- Changes could be done in the next Subscriptions PR indeed

But it's okay to do it here